### ISSUE TITLE
Allow dots as input

### ISSUE THREAD

**Issue by Empty2k12** [liefery/android-signature-view#2]
On my Courier tour I had people trying to input a dot. You have voiced concerns over dots as signatures.

A easy fix would be setting a min amount of `PathDescriptior` actions.

This way people trying to sign with "T. Test" or with names containing ÖÄÜ will be able to sign.

**Comment by taig** [comment]
Yes, it would be nice if it was possible to draw and render a dot. The isEmpty method currently checks (afaik) if at least one line is in the path, which is probably already the behavior we want.
